client/post-editor/editor-title/index.jsx

```diff
@@ -66,6 +66,8 @@ export default React.createClass( {
 		} );
 
 		this.onChange( event );
+
+		event.target.scrollLeft = 0;
```

Comment:
>Is this something that should really be part of `FormTextInput`? Feels brittle to be monkeying with a DOM element from a custom element.

That would depend whether we think browser quirks fall under the scope of behaviors to be implemented in the base form components. I'd personally lean towards keeping them as "pure" as possible such that their purpose is primarily aimed at keeping form styling consistent across the app. That said, an alternative view would be to consider them as normalized components, in which this would help keep behavior consistent across browsers.